worlds/zillion/__init__.py

```diff
@@ -47,7 +47,7 @@ class RomStart(str):
         """
 
     rom_file: RomFile = RomFile(RomFile.copy_to)
-    rom_start: typing.Union[RomStart, bool] = RomStart("retroarch")
+    rom_start: RomStart | bool = RomStart("retroarch")
```

Comment:
is this supposed to be here?

Reply:
I used this to find the problem and to test the test, so I thought it would be appropriate.
Without this, there's nothing to fail that part of the test if it's wrong.

But I can remove it if you want.